
The ident for Seagate 8GB microdrive is
"SEAGATE", "ST1"
hash 0x76dc4190, 0xcfba9599
manfid 0x0111, 0x0000

CF card:
 product info: "SAMSUNG", "04/05/06", "", ""
 manfid : 0x0000, 0x0000
 function 4 (fixed disk)


Signed-off-by: Marcin Juszkiewicz <openembedded@hrw.one.pl>

 drivers/ide/legacy/ide-cs.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.17/drivers/ide/legacy/ide-cs.c
===================================================================
--- linux-2.6.17.orig/drivers/ide/legacy/ide-cs.c	2006-06-18 03:49:35.000000000 +0200
+++ linux-2.6.17/drivers/ide/legacy/ide-cs.c	2006-07-10 23:29:28.696446000 +0200
@@ -402,6 +402,9 @@
 	PCMCIA_DEVICE_PROD_ID12("PCMCIA", "CD-ROM", 0x281f1c5d, 0x66536591),
 	PCMCIA_DEVICE_PROD_ID12("PCMCIA", "PnPIDE", 0x281f1c5d, 0x0c694728),
 	PCMCIA_DEVICE_PROD_ID12("SHUTTLE TECHNOLOGY LTD.", "PCCARD-IDE/ATAPI Adapter", 0x4a3f0ba0, 0x322560e1),
+	PCMCIA_DEVICE_PROD_ID12("SEAGATE", "ST1", 0x87c1b330, 0xe1f30883),	/* Seagate 8GB microdrive */
+	PCMCIA_DEVICE_PROD_ID12("SAMSUNG", "04/05/06", 0x43d74cb4, 0x6a22777d),
+	PCMCIA_DEVICE_PROD_ID12("SMI VENDOR", "SMI PRODUCT", 0x30896c92, 0x703cc5f6),
 	PCMCIA_DEVICE_PROD_ID12("TOSHIBA", "MK2001MPL", 0xb4585a1a, 0x3489e003),
 	PCMCIA_DEVICE_PROD_ID1("TRANSCEND    512M   ", 0xd0909443),
 	PCMCIA_DEVICE_PROD_ID12("WIT", "IDE16", 0x244e5994, 0x3e232852),
